test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -162,6 +164,123 @@ public static void Main()
         }
 
         [Fact]
+        public void Conflicts_are_resolved_when_publishing_a_portable_app()
+        {
+            Conflicts_are_resolved_when_publishing(false);
+        }
+
+        [Fact]
+        public void Conflicts_are_resolved_when_publishing_a_self_contained_app()
+        {
+            Conflicts_are_resolved_when_publishing(true);
+        }
+
+        void Conflicts_are_resolved_when_publishing(bool selfContained, [CallerMemberName] string callingMethod = "")
+        {
+            var targetFramework = "netcoreapp2.0";
+            var rid = selfContained ? EnvironmentInfo.GetCompatibleRid(targetFramework) : null;
```

Comment:
I've tried adding a test for this, and it's getting a bunch of DLLs in the output that shouldn't be copied there.  I think it's because the version of the Microsoft.NETCore.App package that we're using (2.0.0-beta-001834-00) has this logic in it:

```xml
  <ItemGroup Condition="'$(RuntimeIdentifier)' == ''">
    <PackageConflictPlatformManifests Include="$(MSBuildThisFileDirectory)Microsoft.NETCore.App.PlatformManifest.txt" />
  </ItemGroup>
```

So the platform manifest isn't getting picked up for rid-specific shared framework apps.  Has this been updated in newer versions of the package?  What version would have a fix?

@eerhardt @ericstj @weshaggard